API.md

```diff
@@ -0,0 +1,96 @@
+### StorageBuckets API Interface
+
+#### The StorageBucketManager Interface
+```
+[
+  Exposed=(Window,Worker),
+  SecureContext
+] interface StorageBucketManager {
+    [NewObject] Promise<StorageBucket> openOrCreate(USVString name,
```

Comment:
I think we want `DOMString` instead of `USVString` throughout the API. See https://heycam.github.io/webidl/#idl-DOMString

In case Cookie Store was your inspiration... we use `USVString` there the strings there end up in HTTP requests. IIUC, WebIDL used to recommend `USVString` for this use case, whereas it now recommends `ByteString`.

See https://heycam.github.io/webidl/#idl-ByteString and https://heycam.github.io/webidl/#idl-USVString

Reply:
(and also see https://w3ctag.github.io/design-principles/#idl-string-types for TAG guidance on this)

I agree that these indeed should be DOMString. Just keep in mind that you can't represent a DOMString with a std::string or mojo string; i.e. you'd have to make sure to use string16 in mojom and elsewhere outside of blink (although if names will be limited to some subset of 7-bit ascii it won't really matter what type is used in the idl/implementation, so DOMString+std::string should be fine...)

Reply:
Sounds good, thanks for the guidance!